plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -88,7 +88,7 @@ def __init__(self):
         __init__().
         """
         super(PublishBlobsStep, self).__init__(constants.PUBLISH_STEP_BLOBS,
-                                               models.Blob.TYPE_ID)
+                                               constants.BLOB_TYPE_ID)
```

Comment:
It ended up being slightly more work than this, but I got this change done.

Unfortunately, it seems that other non-backwards compatible changes had been made on the platform publish steps since I made this pull request, and so I also had to go and change quite a few other things too.

Fixed.